test/cloudwatchlogs/publish_logs_test.go

```diff
@@ -119,9 +161,9 @@ func TestWriteLogsToCloudWatch(t *testing.T) {
 
 			// ensure that there is enough time from the "start" time and the first log line,
 			// so we don't miss it in the GetLogEvents call
-			time.Sleep(sleepForFlush)
+			time.Sleep(sleepForExtendedFlush)
```

Comment:
Why not just increase `sleepForFlush`? Do we use `sleepForFlush` anywhere else in the code?